lib/devices_api/users/inputs/signup_request_input.ex

```diff
@@ -0,0 +1,29 @@
+defmodule DevicesApi.Users.Inputs.SignupRequestInput do
+  @moduledoc """
+  Create input type.
+  """
+  use DevicesApi.ValueObjectSchema
+
+  import DevicesApi.ChangesetValidation, only: [validate_email: 2]
```

Comment:
Aqui tem uma questão de dependências entre módulos. Temos compile-time e export-time. Vale ler um pouco aqui: https://dashbit.co/blog/speeding-up-re-compilation-of-elixir-projects

Eu faria apenas um alias e chamaria com `|> Changesets.validate_email(:email)`